Codeinwp/tweet-old-post
# Add `Custom Content` for Post message

### Summary

Add `Custom Content` for the Post  message following the described specification in https://github.com/Codeinwp/tweet-old-post-pro/issues/526

Checklist
- [x] Featured locked for Free Users
- [x] E2E test added 

### Screenshots 

#### Free User

![image](https://github.com/Codeinwp/tweet-old-post/assets/17597852/51ea5360-8203-4fdf-87d9-6e93372bb75b)

#### Pro User

https://github.com/Codeinwp/tweet-old-post/assets/17597852/1ab96e59-224c-4f6a-8cb4-35db2967eb92

### Testing

> [!NOTE]
> Test it together with https://github.com/Codeinwp/tweet-old-post-pro/pull/532

1. Install the Pro version to have magic tags enabled.
2. Add a social network and go to `Post Format` tab.
3. Select `Custom Content` for **Share Content** option.
4. You should see some UI changes.
5. Add some content with magic tags in **Message Content**
6. Share the post on the social network and check if the posted message respects the given format.
